redwood/api/src/tasks/syncStarknetState.ts

```diff
@@ -90,6 +91,7 @@ export const importProfile = async (profileId: number) => {
     ),
 
     appealTimestamp: parseTimestamp(profile.appeal_timestamp),
+    appealId: parseBigNumber(profile.appeal_id),
```

Comment:
I guess it doesn't affect the UI since we only show the `appealId` if the `appealTimestamp` is non-null, but I feel like it would be more correct to set this to `null` instead of `0` if the profile has no appeal id?

Reply:
yup agreed, was planning on making this change since I heard back from you about the purpose of nullability in these CachedProfile fields